openapi.yaml

```diff
@@ -1453,84 +1405,21 @@ paths:
         - name: amount
           in: query
           description: The amount of the asset to convert
-          required: true
           schema:
             type: string
+            default: 1
       responses:
         '200':
-          description: The rate and converted value
+          description: The rate and amount of the toAssetId
           content:
             application/json:
               schema:
                 title: ConvertResponse
                 type: object
                 properties:
-                  fromAssetId:
-                    description: The asset to convert from
-                    type: string
-                  toAssetId:
-                    description: The asset to convert to
-                    type: string
                   amount:
-                    description: The amount of the from asset to convert
+                    description: The amount of the to asset using the conversion rate
                     type: string
                   rate:
                     description: The rate used for the converstion
                     type: string
-                  timestamp:
-                    description: The timestamp of the rate used
-                    type: string
-                    format: date-time
-                  result:
```

Comment:
We need the result object as well right?

Reply:
That was a old field which we are no longer going to have now.